src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -477,6 +504,13 @@ public void End()
 			handler?.Invoke(this, EventArgs.Empty);
 			Ended = null;
 
+			if (SpanTimings.ContainsKey(("app", null)))
+				SpanTimings[("app", null)].IncrementTimer(SelfDuration);
+			else
+				SpanTimings.TryAdd(("app", null), new SpanTimer(SelfDuration));
+
+			_breakdownMetricsProvider?.CaptureTransaction(this);
```

Comment:
Could breakdown metrics be captured by subscribing to the `Transaction.End` event in `Tracer`? It would avoid the need to pass the breakdown metrics provider to each transaction to call in the `End()` method.

Reply:
Yes, we can do that. In addition to that we also don't need to pass `_breakdownMetricsProvider` to spans - so I also removed it from the span `.ctor`.